File: server/src/Bananagrams.Api/ViewModels/Users/UserViewModel.cs
namespace Bananagrams.Api.ViewModels.Users;

public class UserViewModel
{
    public int Id { get; set; }
    public string EmailAddress { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string Username { get; set; }
    public int TotalSolved { get; set; }
    public int TotalAttempts { get; set; }
}

File: server.java/bananagrams/src/main/java/coe/bananagrams/viewModel/user/UserViewModel.java
package coe.bananagrams.viewModel.user;

import coe.bananagrams.dto.gameUser.GameUserDto;
import lombok.*;

import java.util.Date;
import java.util.List;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserViewModel {
    private int id;
    private Date dateCreated;
    private String firstName;
    private String lastName;
    private String username;
}
